README.md

```diff
@@ -11,18 +11,39 @@ _Variables in bold are required._
 | AWS_ECR_REPOSITORY          |                    | The AWS ECR repository. **This is only required as GitHub repository secret.** |
 | AWS_REGION                  |                    | The AWS region. **This is also required as GitHub repository secret.**         |
 | AWS_SECRET_ACCESS_KEY       |                    | The AWS access key. **This is also required as GitHub repository secret.**     |
-| CONCURRENCY                 | `16`               | The maximum concurrency when indexing CARs.                                    |
+| CONCURRENCY                 | `8`                | Concurrent batch inserts of blocks.                                            |
+| BLOCKS_BATCH_SIZE           | `10`               | Batch size for blocks ops (insert, publish). 10 is max for SQS, 25 is max for Dynamo |
 | DECODE_BLOCKS               |                    | Set to `true` to decode non raw block information and store then in DynamoDB   |
-| DYNAMO_BLOCKS_TABLE         | `blocks`           | The DynamoDB table where store CIDs informations to.                           |
-| DYNAMO_CARS_TABLE           | `cars`             | The DynamoDB table where store CAR files informations to.                      |
-| DYNAMO_MAX_RETRIES          | 3                  | DynamoDB max attempts in case of query failure.                                |
-| DYNAMO_RETRY_DELAY          | 100                | DynamoDB delay between attempts in case of failure, in milliseconds.           |
-| S3_MAX_RETRIES              | 3                  | S3 max attempts in case of failure.                                            |
-| S3_RETRY_DELAY              | 100                | S3 delay between attempts in case of failure, in milliseconds.                 |
+| DYNAMO_BLOCKS_TABLE         | `v1-blocks`        | The DynamoDB table where store CIDs informations to.                           |
```

Comment:
How will this migration work?

Reply:
we will use the same migration script of the last time
https://github.com/ipfs-elastic-provider/ipfs-elastic-provider-infrastructure/tree/main/terraspace/app/stacks/indexing/bucket-mirroring-trigger

it gets the cars from the s3 bucket and it pushes them to the indexer queue